wiki/middleware.py

```diff
@@ -0,0 +1,27 @@
+import threading
+
+_thread_locals = threading.local()
+
+
+def get_current_request():
+    """Return the request associated with the current thread."""
+    return getattr(_thread_locals, 'request', None)
```

Comment:
We already have the `RequestCache` middleware which provides this functionality so lets not introduce another into edx-platform. You should try to import that first and if available use it instead.

https://github.com/edx/edx-platform/blob/master/common/djangoapps/request_cache/middleware.py